packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -44,24 +45,24 @@
   }
 
   &:disabled {
-    border-color: transparent;
+    border-color: var(--ColorBorderPrimaryDisabled);
     background-color: var(--ColorBackgroundPrimaryDisabled);
     cursor: not-allowed;
   }
 
   & + & {
-    margin-inline-start: $spacing-s;
+    margin-inline-start: var(--GutterMarginInlineStartPrimary);
```

Comment:
I would feel okay calling this `--Gutter` or `--GutterMargin`. `margin-inline-start` is the method we use, but the value could be applied to a flex or grid solution in the future.

While all of these styles apply to Primary buttons, this gutter is non-specific, so we could safely drop that as well.

In future modules, it might be beneficial for us to treat all variant CSS as such rather than build it into Root's default. That way, it's clear which properties are dynamic.